src/Illuminate/Console/Events/CommandTerminating.php

```diff
@@ -0,0 +1,36 @@
+<?php
+
+namespace Illuminate\Console\Events;
+
+use Symfony\Component\Console\Input\InputInterface;
+
+class CommandTerminating
+{
+    /**
+     * The command name.
+     *
+     * @var string
+     */
+    public $command;
+
+    /**
+     * The console input.
+     *
+     * @var string
+     */
+    public $input;
+
+    /**
+     * Create a new event instance.
+     *
+     * @param  string $command
+     * @param \Symfony\Component\Console\Input\InputInterface $input
+     * @param int $exitCode
+     * @return void
+     */
+    public function __construct($command, InputInterface $input, $exitCode)
+    {
+        $this->command = $command;
+        $this->input = $input;
+    }
```

Comment:
`$exitCode` should also be declared as a public property and set in the constructor.